pkg/hyperkit/constants.go

```diff
@@ -2,7 +2,7 @@ package hyperkit
 
 const (
 	DriverName    = "hyperkit"
-	DriverVersion = "0.12.14"
+	DriverVersion = "0.14.0"
```

Comment:
> The change no longer starting the VM after create is
non-backwards-compatible, that's a good opportunity to switch
to 0.14 for our versioning.

Then why not `0.13.0` instead going to `0.14.0`  :)

Reply:
Eh, this is not even intentional, I'm used to even number -> stable release, odd number -> development release. In our case, 0.13 is fine :)